### DISCUSSION TITLE
Can I freely use Saleor Logo as cover for the doc of this plugin I created ?

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#10643] [General] [answered]
Hi,
in order to increase even more the development experience I created a plugin to connect Saleor products and variants with [DatoCMS](https://www.datocms.com/).

You can fined and install it from: [https://www.datocms.com/marketplace/plugins/i/datocms-plugin-saleor](https://www.datocms.com/marketplace/plugins/i/datocms-plugin-saleor)

And the repo is here [https://github.com/DidoMarchet/datocms-plugin-saleor](https://github.com/DidoMarchet/datocms-plugin-saleor)

As you can see I use you logo as cover.
I want to ask to you I can freely use that.

Really thanks in advance and kind regards,

Davide

**Comment by jwgraves2** [comment, marked as answer]
Hi Davide, 
We're excited to hear about the Dato integration you've built! 
We approve of your use of Saleor's logo. We ask only that the logo remains unaltered and that the page clearly identifies the plugin as built and maintained by the community. 

Best, 
Wilton

**Reply by DidoMarchet** [reply]
Hi @jwgraves2 ,thanks !
